cmd/podman/pod_restart.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	podRestartFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "restart all pods",
+		},
+		LatestFlag,
+	}
+	podRestartDescription = `Restarts one or more pods. The pod ID or name can be used.`
+
+	podRestartCommand = cli.Command{
+		Name:                   "restart",
+		Usage:                  "Restart one or more pods",
+		Description:            podRestartDescription,
+		Flags:                  podRestartFlags,
+		Action:                 podRestartCmd,
+		ArgsUsage:              "CONTAINER [CONTAINER ...]",
```

Comment:
Should this be Container or Pod?   Also per other PR's, I think we've a way to specify that either the ID or Name can be used.  But if saying POD is what's been settled on rather than 'PODID|PODNAME' or some such, that's fine.